### ISSUE TITLE
Move backwards, strafe left, strafe right cause crossing walls.

### ISSUE THREAD

**Issue by microaaron** [x3dom/x3dom#1186]
[https://twitter.com/microaaron/status/1471970864958636032](https://twitter.com/microaaron/status/1471970864958636032)

**Comment by microaaron** [comment]
@andreasplesch They alse cause rushing out of the map.
[https://twitter.com/microaaron/status/1471980966562336772](https://twitter.com/microaaron/status/1471980966562336772)

**Comment by microaaron** [comment]
@andreasplesch Well, [checking front for collisions](https://github.com/x3dom/x3dom/blob/master/src/nodes/Navigation/modes/DefaultNavigation.js#L78) is effective only for moveForward.

**Comment by andreasplesch** [comment]
Yes, that is where the check is. It may be possible to check the other
directions in the same way but it will impact performance.

---on the phone---
